openpype/hosts/maya/api/lib.py

```diff
@@ -2199,6 +2199,8 @@ def reset_frame_range(playback=True, render=True, fps=True):
         render (bool, Optional): Whether to set the maya render frame range.
             Defaults to True.
         fps (bool, Optional): Whether to set scene FPS. Defaults to True.
+        instances (bool, Optional): Whether to update publishable instances.
+            Defaults to True.
```

Comment:
```suggestion
        instances (bool, Optional): Whether to update publishable instances.
            Defaults to True. Even when True this will only reset instances if
            `update_publishable_frame_range` is also enabled in project settings.
```

Should we clarify that even when True it only does so if also enabled in settings?

Reply:
@friquette can we get this in?